user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_install_setup.md

```diff
@@ -0,0 +1,38 @@
+---
+uid: Data_Aggregator_install_setup
+---
+
+# Data Aggregator installation and setup
+
+The Data Aggregator DxM does not necessarily have to be installed on a server running a DataMiner Agent. You can also install it on a standalone server.
+
+## Prerequisites
+
+- DataMiner 10.2.12 or higher must be installed on the DataMiner Agents you want to use Data Aggregator with.
+
+  > [!NOTE]
+  > If you want to use queries with [ad hoc data sources](xref:Configuring_GQI_feeds#get-ad-hoc-datacustom-data), you will need DataMiner 10.3.3/10.4.0 or higher.
+
+- The CoreGateway DxM must be installed on the DataMiner Agents you want to use Data Aggregator with. To install this DxM, you can use the [DataMiner Cloud Pack](xref:CloudPackages). The DMAs do not have to be connected to dataminer.services.
+
+## Installing the Data Aggregator DxM
+
+Download the installation package for the Data Aggregator DxM from the [Dojo Downloads page](https://community.dataminer.services/downloads/) and install it.
+
+## Configuring the Data Aggregator settings
```

Comment:
If this is the settings file where the other examples should belong in, it wouldn't be bad to refer to here